app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -101,7 +101,7 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     dateOfBirthEditText.imeOptions += EditorInfo.IME_ACTION_NEXT
     dateOfBirthEditText.setOnEditorActionListener { _, actionId, _ ->
       // When date is empty, this will move focus to age field and colony field otherwise.
-      if (!dateOfBirthEditText.text.isBlank() && actionId == EditorInfo.IME_ACTION_NEXT) {
+      if (!dateOfBirthEditText.text!!.isBlank() && actionId == EditorInfo.IME_ACTION_NEXT) {
```

Comment:
Update: wow this is so weird, `EditText#text` won't ever be null because `TextView#setText()` converts nulls into empty strings. But `AppCompatEditText` can return null if the text is not editable. The fun part is that text in `EditText` is always editable. Why did they do this :S

Reply:
Exactly. No idea why this was added, but the app wouldn't compile anymore. 💯